### PR TITLE
Examples copy update

### PR DESCRIPTION
**Pull request by mollieru**
Updates example content for the following components:
> tables
> checkboxes
> radio buttons
> grids
> accordions
> sidenav
> typesetting

### CODE REVIEW

File: _elements/tables.md [hunk 1]
```diff
@@ -34,20 +34,15 @@ lead: Tables show tabular data in columns and rows.
         <td>$3.78</td>
       </tr>
       <tr>
-        <td scope='row'>Key Lime Pie</td>
-        <td>Made of Key lime juice, egg yolks, and sweetened condensed milk in a pie crust</td>
-        <td>$1.99</td>
+        <td scope='row'>Declaration of Sentiments</td>
+        <td>MadeA document written during the Seneca Falls Convention outlining the rights that American women should be entitled to as citizens.</td>
```
Comment (maya) [inline _elements/tables.md]: MadeA - is this right?

File: _elements/tables.md [hunk 2]
```diff
@@ -12,21 +12,21 @@ lead: Tables show tabular data in columns and rows.
...
       <tr>
-        <td scope='row'>Blueberry Pie</td>
-        <td>Composed of blackberry filling, usually in the form of either blackberry jam</td>
-        <td>$4.25</td>
+        <td scope='row'>Bill of Rights</td>
+        <td>The first ten amendments of the U.S. Constitution guaranteeing rights and freedoms.</td>
+        <td>1791</td>
       </tr>
       <tr>
         <td scope='row'>Blackberry Pie</td>
```
Comment (maya) [inline _elements/tables.md]: @mollieru is this supposed to be pie?

**Comment by maya** [comment]
@mollieru the new content on the grids makes them uneven. Is this desired?

screen shot 2015-09-23 at 1 23 23 pm

**Comment by mollieru** [comment]
@maya will fix those things. stay tuned.

**Comment by mollieru** [comment]
@maya lmk if those fixes work. thank you for catching those!

**Comment by maya** [comment]
:+1:

**Pull request merged**